### PR TITLE
Update AreaSeries.cs

### PR DESCRIPTION
**Pull request by less0**
Implemented the calculation of baseline values if Points2 is empty.

### CODE REVIEW

**Comment by objorke** [comment]
Please create an issue first. :-)
See also https://github.com/oxyplot/oxyplot/blob/develop/CONTRIBUTING.md

File: Source/OxyPlot/Series/AreaSeries.cs [hunk 1]
```diff
@@ -118,7 +119,9 @@ protected List<DataPoint> ActualPoints2
         {
             get
             {
-                return this.ItemsSource != null ? this.itemsSourcePoints2 : this.points2;
+                return this.ItemsSource != null 
+                    ? this.itemsSourcePoints2 
+                    : (this.points2.Any() ? this.points2 : this.Points.Select(p => new DataPoint(p.X, ConstantY2)).ToList());
```
Comment (tibel) [inline Source/OxyPlot/Series/AreaSeries.cs]: Only use ToList() for the `Select()` clause as `this.points2` is a list already.
Also I think the points should be cached in `itemsSourcePoints2` similar to the implementation of DataPointSeries.

**Comment by objorke** [comment]
I think two points should be sufficient for the second point set using "ConstantY2" values.

**Comment by objorke** [comment]
Reference: #662

**Comment by objorke** [comment]
I squashed your commits in https://github.com/objorke/oxyplot/tree/bugfix/areaseries-pauls-solution
and created a slightly alternative solution in https://github.com/objorke/oxyplot/tree/bugfix/areaseries-constant-baseline
See PR in #701 - includes examples, updated change log and everything in 1 commit

**Comment by objorke** [comment]
I merged #701

**Pull request closed without merging**